src/main/io/serial_cli.c

```diff
@@ -737,6 +739,38 @@ static void cliCMix(char *cmdline)
 #endif
 }
 
+static void cliRxCalibration(char *cmdline)
```

Comment:
Using get/set mechanism is probably cleaner that adding new command. What do you think, @hydra? 

Also there is probably no need to support calibration of all 18 channels (high channel count radios don't need it and there is little use of calibration for AUX channels). Supporting AETR (or maybe AETR1234) should be enough.